xml/art_migration.xml

```diff
@@ -385,22 +371,60 @@
      </listitem>
      <listitem>
       <para>
-       Otherwise reformat the disk using partitioning to match the current
-       configuration.
+       Otherwise, if installing manually or using a proprietary tool for
+       provisioning, reformat the primary partition (preserving the overall
+       partitioning scheme based on the data model) and install the &sls; 12 SP3
+       operating system.
       </para>
      </listitem>
      <listitem>
       <para>
-       Copy the entire <filename>~stack</filename> directory from the original
-       deployer:
+       Update &compnode;s' repository references as appropriate to point to the seed node.
+      </para>
+      <itemizedlist>
+       <listitem>
+        <para>
+         On each &slsa; compute node, if the file <filename>/etc/zypp/repos.d/SLES12-SP3-12.3-0.repo</filename> exists (pointing to the cobbler deployment repo on the original deployer), delete it:
```

Comment:
I don't think it's very important, but I am wondering why you didn't break the lines in this section as you did in previous ones? Those lines are rather long now :smile:.